lms/djangoapps/shoppingcart/migrations/0026_migrate_invoices.py

```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+from decimal import Decimal, ROUND_DOWN
+from south.v2 import DataMigration
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Remember to use orm['appname.ModelName'] rather than "from appname.models..."
+        for invoice in orm.Invoice.objects.extra(
+            select={
+                'qty': 'SELECT COUNT(*) FROM shoppingcart_courseregistrationcode '
+                       'WHERE shoppingcart_courseregistrationcode.invoice_id = shoppingcart_invoice.id'
+            }
+        ).all():
```

Comment:
Please update the comments to explain this query -- it took me a few minutes to figure out what this was doing.

Also, small style nitpick: this is a fairly complicated line.  I think it would be easier to read if you assigned the queryset to a variable:
```
queryset = orm.Invoice.objects.extra(...).all()
for invoice in queryset:
    # ...
```